packages/graphics/src/Graphics.js

```diff
@@ -1033,7 +1033,16 @@ export default class Graphics extends Container
     _calculateBounds()
     {
         this.finishPoly();
-        const lb = this.geometry.bounds;
+
+        const geometry = this.geometry;
+
+        // skipping when graphics is empty, like a container
+        if (!geometry.graphicsData.length)
+        {
+            return;
+        }
+
+        const { minX, minY, maxX, maxY } = geometry.bounds;
 
         this._bounds.addFrame(this.transform, lb.minX, lb.minY, lb.maxX, lb.maxY);
```

Comment:
Also, need to fix this line to remove `lb`

Reply:
yes, I noticed this, thanks.